include/enums.h

```diff
@@ -3011,4 +3011,9 @@ enum BattleSubmenuStatus {
     BATTLE_SUBMENU_STATUS_NO_TARGETS_2      = -2, // TODO: determine difference, probably uses a different error message
 };
 
+enum DictionaryIndex{
+    DICTIONARY_KEY   = 0,
+    DICTIONARY_VALUE = 1,
+};
```

Comment:
Unsure if this is really needed, would be nice if we could do like
```c
s32 key = defenseTable[0];
s32 value = defenseTable[1];

// ...

defenseTable += 2;
```
but idk if that would match in lookup_defense?

Reply:
Actually it might be cool to do
```suggestion
    DICTIONARY_ENTRY_SIZE,
};
```
at the bottom and then increment defenseTable by DICTIONARY_ENTRY_SIZE (2)